modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/servlet/taglib/clay/BlogsEntryVerticalCard.java

```diff
@@ -133,12 +138,17 @@ public String getSubtitle() {
 			PortalUtil.getHttpServletRequest(renderRequest),
 			System.currentTimeMillis() - modifiedDate.getTime(), true);
 
-		return LanguageUtil.format(
-			_resourceBundle, "x-ago-by-x",
-			new Object[] {
-				modifiedDateDescription,
-				HtmlUtil.escape(_blogsEntry.getUserName())
-			});
+		Format shortDateFormat = 
+				getSimpleDateFormat("MM/dd/yy 'at' h:mm a", LocaleUtil.getDefault(), TimeZoneUtil.getDefault());
+		String startTag = "<span title=\"" + shortDateFormat.format(modifiedDate) + "\">";
```

Comment:
I think we can add `class="lfr-portal-tooltip"` to show Liferay's tooltip style instead of relying on the browser's tooltip.